install-dependencies.sh

```diff
@@ -121,6 +116,17 @@ fedora_python3_packages=(
     python3-six
 )
 
+pip_packages=(
+    'geomet<0.3,>=0.1'
+    boto3
+    pytest
+    pytest-asyncio
+    redis
+    aiohttp
+    traceback-with-variables
+    scylla-api-client
+)
+
```

Comment:
I don't understand the split between pip_packages and _scylla_pip_packages (or why that has an extra underscore).

Reply:
Please see commits msg
In short, I want to use the pip_packages term for the packages will be installed by pip which is not the case with the current assignment,
tbh - unless I missed something - I'm not sure why this list which is only used by scylla-python3 is here - @syuu1228 can't we move it to a conf file/env on the other project?